modules/lib/lib-project/src/main/resources/lib/xp/project.js

```diff
@@ -0,0 +1,191 @@
+/**
+ * Functions to manage Content Projects.
+ *
+ * @example
+ * var projectLib = require('/lib/xp/project');
+ *
+ * @module project
+ */
+
+/* global __ */
+
+function required(params, name) {
+    var value = params[name];
+    if (value === undefined) {
```

Comment:
Codacy found an issue: [Unexpected use of undefined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6044642)